### PR TITLE
Update rpgffi 0.3.3 pg 10.1

### PR DESCRIPTION
**Pull request by posix4e**

### CODE REVIEW

**Comment by allenap** [comment]
I've tried this with PostgreSQL 9.6 and 10.1 via Docker on a Mac, and all's good so far, meaning the tests pass. As yet I haven't had an opportunity to do anything more involved.

**Comment by posix4e** [comment]
@solidsnack Looks good to me.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "jsoncdc"
-version = "0.0.11"
+version = "0.0.12"
```
Comment (solidsnack) [inline Cargo.toml:3]: This has to be updated in `jsoncdc.control` and `META.json` as well.

File: Makefile [hunk 2]
```diff
@@ -41,7 +41,7 @@ ifeq ($(HAZRUST),yes)
 .PHONY: jsoncdc.so
 jsoncdc.so:
 	cargo rustc $(FEATURES) --release -- $(LINK_FLAGS)
-	cp target/release/libjsoncdc.* $@
+	cp target/release/libjsoncdc.*so $@
```
Comment (solidsnack) [inline Makefile]: Breaks on my Mac (where it is `libjsoncdc.dylib`). What lead to this change?

**Review by solidsnack — CHANGES_REQUESTED** [review]
Outside the two comments, everything looks good and I'm ready to approve. Let me know more about what lead to the change on `Makefile:44` and maybe I can come up with something.

File: Makefile (hunk 2 above)
Comment (posix4e) [reply Makefile]: Ahh, this no longer works on linux as two files match libjsoncdc.*

**Comment by posix4e** [comment]
@solidsnack I made the change a bit more restrictive, tell me if that meets your fancy

**Comment by posix4e** [comment]
Before we merge we should squash IMHO

File: Makefile [hunk 3]
```diff
@@ -41,7 +41,7 @@ ifeq ($(HAZRUST),yes)
 .PHONY: jsoncdc.so
 jsoncdc.so:
 	cargo rustc $(FEATURES) --release -- $(LINK_FLAGS)
-	cp target/release/libjsoncdc.*[so\|dll] $@
+	cp target/release/libjsoncdc.[so\|dll] $@
```
Comment (solidsnack) [inline Makefile]: On Mac, it's `dylib` but I am honestly not sure this style of pattern is the way to go. In #80 there's a patch to this branch that uses GNU Make wildcards.

**Review by solidsnack — CHANGES_REQUESTED** [review]

**Pull request merged**